src/platforms/javascript/common/session-replay/understanding-sessions.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Understanding Sessions
+sidebar_order: 4300
+notSupported:
+  - javascript.cordova
+  - javascript.electron
+  - javascript.wasm
+description: 'Learn about customizing sessions with the Session Replay SDK.'
+---
+
+## Default Session Initialization
+
+By default, Replay will immediately start a session in either `session` or `buffer` mode, depending on your `replaysSessionSampleRate` and `replaysOnErrorSampleRate`.
+
+When Replay is initialized, we check the `replaysSessionSampleRate`.
+If it is sampled, it will start recording in `session` mode and begin sending Replay data immediately.
+
+Else, if `replaysOnErrorSampleRate > 0`, it will start recording in `buffer` mode.
+In this mode, when an error occurs, `replaysOnErrorSampleRate` is used to check if it should be sampled. If it is sampled, then the Replay will be uploaded to Sentry and continue recording normally.
+
+When both `replaysSessionSampleRate` and `replaysOnErrorSampleRate` are `0`, recording will _not_ start.
+In this case, you can manually start recording.
+
+## Session Mode
+
+In `session` mode, Replay will continuously record and send data to Sentry. After 15 minutes of user inactivity, or a maximum duration of 60 minutes, the session will end and a new session will be initialized based on the rules from the section above.
+
+## Buffer Mode
+
+In `buffer` mode, Replay will continuously record data, but only keep up to the last 60 seconds of data in memory. When an error occurs, `replaysOnErrorSampleRate` is checked and if it is sampled, the replay will be uploaded to Sentry and continue recording normally. After 15 minutes of user inactivity, or a maximum duration of 60 minutes, the session will end and the replay will stop.
+
+## Manually Starting Recording
+
+If both `replaysSessionSampleRate` and `replaysOnErrorSampleRate` are `0`, recording will _not_ start. In this case, you can manually start recording:
+
+```javascript
+// This initializes Replay without starting any session
+const replay = new Sentry.Replay({
+  replaysSessionSampleRate: 0,
+  replaysOnErrorSampleRate: 0,
+});
+
+// This starts in `session` mode, regardless of sample rates
+replay.start();
+
+// OR
+
+// This starts in `buffer` mode, regardless of sample rates
+replay.startBuffering();
+```
+
+## Manually Stopping Recording
+
+You can always stop a running session:
+
+```javascript
+await replay.stop();
+```
+
+This will flush any pending recording data, stop the replay, and end the session.
+
+## Manually Flushing Recording Data
+
+You can flush any pending recording data:
+
+```javascript
+await replay.flush();
+```
+
+When in `session` mode, this will upload any pending recording data to Sentry.
+When in `buffer` mode, this will upload any pending recording data to Sentry and then continue recording, the same as if an error is sampled due to `replaysOnErrorSampleRate`.
```

Comment:
```suggestion
In `session` mode, this will upload any pending recording data to Sentry. In `buffer` mode, this will upload any pending recording data to Sentry and then continue recording, the same as when an error is sampled with `replaysOnErrorSampleRate`.
```